docs/source/api/apollo-server.mdx

```diff
@@ -600,16 +600,46 @@ The `start` method triggers the following actions:
+
+- Calls and awaits all [`drainServer` plugin handlers](../integrations/plugins-event-reference/#drainserver). These should generally:
+  * Stop listening for new connections
+  * Closes idle connections (i.e., connections with no current HTTP request)
+  * Closes active connections whenever they become idle
+  * Waits for all connections to be closed
+  * After a grace period, if any connections remain active, forcefully close them.
+  If you're using the batteries-included `apollo-server` package, it does this by default. (You can configure the grace period with the [`stopGracePeriodMillis` constructor option](#stopgraceperiodmillis).) Otherwise, you can use the [drain HTTP server plugin](./plugin/drain-http-server/) to drain your HTTP server.
+- Transitions the server to a state where it will not start executing more GraphQL operations.
+- Calls and awaits all [`serverWillStop` plugin handlers](../integrations/plugins-event-reference/#serverwillstop) (including the [usage reporting plugin](./plugin/usage-reporting/)'s handler, which sends a final usage report to Apollo Studio).
+- If your server is a [federated gateway](https://www.apollographql.com/docs/federation/gateway/), `stop` also stops gateway-specific background activities, such as polling for updated service configuration.
+
+This method takes no arguments. You should only call it after [`start()`](#start) returns successfully (or [`listen()`](#listen) if you're using the batteries-included `apollo-server` package).
+
+In some circumstances, Apollo Server calls `stop` automatically when the process receives a `SIGINT` or `SIGTERM` signal. See the [`stopOnTerminationSignals` constructor option](#stoponterminationsignals) for details.
+
+
 ## `executeOperation`
 
-The async `executeOperation` method is used primarily for [testing GraphQL operations](../testing/testing/#testing-using-executeoperation) through the server's request pipeline without going through a full HTTP operation.
+The async `executeOperation` method is used primarily for [testing GraphQL operations](../testing/testing/#testing-using-executeoperation) through Apollo Server's request pipeline _without_ sending an HTTP request.
+
+```js
+const result = await server.executeOperation({
+  query: 'query SayHelloWorld($name: String) { hello(name: $name) }',
+  variables: { name: 'world' },
+});
+```
 
 The `executeOperation` method takes two arguments:
-* The first argument is a configuration options object
-* The second argument will be passed in to the current `ApolloServer` instance's `context` function
+* The first argument is a configuration options object.
+  * Supported options are listed in the table below.
+* The second argument is passed to `ApolloServer`'s `context` function.
+  * This is helpful when [testing](../testing/testing/) that your server's `context` is correctly gathering the values needed from a request.
```

Comment:
@StephenBarlow How does this phrasing feel? I wanted to keep this point brief, but I can expand this if it feels too vague.

Reply:
That looks fine to me!